<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>1141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="org">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>20</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Knee</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="p1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Output">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Sharp">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>220</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string> enhancement</string>
    </property>
   </widget>
   <widget class="QLabel" name="p2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="p2_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>730</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="org_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Output_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>580</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="p1_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="open_2">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>130</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lung</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="adjust">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>340</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adjust</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Knee">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>570</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Lung">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>660</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
